doc/website/blog/2022-09-05-announcing-migration-execution.mdx

```diff
@@ -0,0 +1,390 @@
+structure: `version_[name].sql`, with the `name` being optional. `version` can be an arbitrary string. Migration files
+are lexicographically sorted by filename.
+
+<Tabs
+    defaultValue="tree"
+    values={[
+        {label: 'Tree', value: 'tree'},
+        {label: '1_initial.sql', value: '1'},
+        {label: '2_second.sql', value: '2'},
+        {label: '3_second.sql', value: '3'},
+        {label: 'atlas.sum', value: 'sum'},
+]}>
+<TabItem value="tree">
+
+```shell
+↪ tree .
+.
+├── 1_initial.sql
+├── 2_second.sql
+├── 3_third.sql
+└── atlas.sum
+
+0 directories, 4 files
+```
+
+</TabItem>
+
+<TabItem value="1">
+
+```sql
+-- add new schema named "my_schema"
+CREATE DATABASE `my_schema`;
+-- create "tbl" table
+CREATE TABLE `my_schema`.`tbl` (`col` int NOT NULL);
+```
+
+</TabItem>
+
+<TabItem value="2">
+
+```sql
+ALTER TABLE `my_schema`.`tbl` ADD `col_2` TEXT;
+```
+
+</TabItem>
+
+<TabItem value="3">
+
+```sql
+CREATE TABLE `tbl_2` (`col` int NOT NULL);
+```
+
+</TabItem>
+
+<TabItem value="sum">
+
+```text
+h1:cD9kOv5VDRLrKVZ0pM4CxAlhH6mgE8PQLpUeuIMKDcs=
+1_initial.sql h1:SrFyOe0eg5WnE96GH3TtAt6046sfrOK4YKZYBlYr1SA=
+2_second.sql h1:FPzwV+MzwyCss7SASZtyafXiYc9Un5bzlcc3u7MxLJU=
+3_third.sql h1:uD+xDcA3Q+gHqwca2ZBDAHWYvC2eiUcwr1IgMsN0Q6c=
+
+```
+
+</TabItem>
+
+</Tabs>
+
+If you want to follow along, you can simply copy and paste the above files in a folder on your system. Make sure you
+have a database ready to work on. You can start an ephemeral docker container with the following command:
+
+```shell
+# Run a local mysql container listening on port 3306.
+docker run --rm --name atlas-apply --detach --env MYSQL_ROOT_PASSWORD=pass -p 3306:3306 mysql
+```
+
+### Apply Migrations
+
+In order to apply migrations you need to have the Atlas CLI in version v0.7.0 or above. Follow the
+[installation instructions](https://atlasgo.io/getting-started#installation) if you have no version installed yet.
+
+Now, to apply the first migration of our migration directory, we call `atlas migrate apply` and pass in some
+configuration parameters.
+
+```shell
+atlas migrate apply 1 \
+  --dir "file://migrations" \
+  --url "mysql://root:pass@localhost:3306/"
+```
+
+```shell
+Migrating to version 1 (1 migrations in total):
+
+  -- migrating version 1
+    -> CREATE DATABASE `my_schema`;
+    -> CREATE TABLE `my_schema`.`tbl` (`col` int NOT NULL);
+  -- ok (17.247319ms)
+
+  -------------------------
+  -- 18.784204ms
+  -- 1 migrations
+  -- 2 sql statements
+
+```
+
+### Migration Status
+
+Atlas saves information about the database schema revisions (applied migration versions) in a special table called
+`atlas_schema_revisions`. Since in our example above the connection was schema-less (we call that `Database Realm`),
+Atlas put the revision table into its own schema called `atlas_schema_revisions` as well. For a schema bound connection
```

Comment:
IMO introducing the internal concept of a Realm isn't useful for this discussion
```suggestion
`atlas_schema_revisions`. In the example above we connected to the database without specifying which schema to operate against.
For this reason, Atlas created the revision table in a new schema called `atlas_schema_revisions`. For a schema bound connection
```